run.go

```diff
@@ -0,0 +1,103 @@
+package rest_data_archiver
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/silinternational/rest-data-archiver/alert"
+	"github.com/silinternational/rest-data-archiver/aws"
+	"github.com/silinternational/rest-data-archiver/internal"
+	"github.com/silinternational/rest-data-archiver/restapi"
+)
+
+func Run(configFile string) error {
+	log.SetOutput(os.Stdout)
+	log.SetFlags(0)
+	log.Printf("Archive started at %s", time.Now().UTC().Format(time.RFC1123Z))
+
+	appConfig, err := internal.LoadConfig(configFile)
+	if err != nil {
+		msg := fmt.Sprintf("Unable to load config, error: %s", err)
+		log.Println(msg)
```

Comment:
No big deal, but I suppose you could factor these three lines out and 
`return logAndEmail(msg, appConfig.Alert)`

(since you do this at least 3 places)